include/nuttx/spinlock.h

```diff
@@ -105,28 +105,7 @@ typedef uint8_t spinlock_t;
  *
  ****************************************************************************/
 
-#if defined(CONFIG_ARCH_HAVE_TESTSET)
 spinlock_t up_testset(volatile FAR spinlock_t *lock);
```

Comment:
could you please fix a comment near `spin_islocked`? and modify to `/* bool spin_islocked(FAR spinlock_t *lock); */`

Reply:
Done.